test/introspection/ERC165Base.ts

```diff
@@ -16,4 +17,42 @@ describe('ERC165Base', function () {
   describeBehaviorOfERC165Base(async () => instance, {
     interfaceIds: [],
   });
+
+  describe('__internal', () => {
+    describe('#_supportsInterface(bytes4)', () => {
+      it('returns whether interface ID is supported', async () => {
+        const interfaceId = ethers.utils.randomBytes(4);
+
+        expect(await instance.callStatic.__supportsInterface(interfaceId)).to.be
+          .false;
+        await instance.__setSupportsInterface(interfaceId, true);
+        expect(await instance.callStatic.__supportsInterface(interfaceId)).to.be
+          .true;
+      });
+    });
+
+    describe('#_setSupportsInterface(bytes4,bool)', () => {
+      it('updates support status forgiven interface', async () => {
```

Comment:
`it('updates support status for given interface', async () => {`

Reply:
forgive the error